esp/esp/web/util/latex.py

```diff
@@ -46,6 +47,7 @@
 
 TEX_TEMP = tempfile.gettempdir()
 TEX_EXT  = '.tex'
+devnull_sentinel = object()
```

Comment:
Made it private. If a caller wants to, say, have ``stdout`` go to a file but ``stderr`` go to ``devnull``, the caller should explicitly open a handle to ``devnull``, pass that for ``stderr``, and ensure proper file closing, rather than use the sentinel.